src/schemathesis/cli/output/default.py

```diff
@@ -224,25 +224,29 @@ def display_statistic(context: ExecutionContext, event: events.Finished) -> None
     total = event.total
     if event.is_empty or not total:
         click.secho("No checks were performed.", bold=True)
-        return
 
+    if total:
+        display_checks_statistics(total)
+
+    if context.cassette_file_name:
+        click.echo()
+        category = click.style("Network log", bold=True)
+        click.secho(f"{category}: {context.cassette_file_name}")
+
+    if context.junit_xml_file:
+        display_section_name(f"generated xml file {context.junit_xml_file}", separator="-")
```

Comment:
"generated" -> "Generated" to be more consistent with the message above.
"xml" -> "XML"

Reply:
Also I think it will be better to have this line without separators around it, but just a single line like "Network log" above